src/test/scala/org/hammerlab/guacamole/main/GeneratePartialFasta.scala

```diff
@@ -77,15 +80,6 @@ object GeneratePartialFasta extends Logging {
       })
     })
     writer.close()
-    Common.progress("Wrote: %s".format(args.output))
-  }
-
-  private def printUsage() = {
```

Comment:
It was unclear to me how best to use it; args4j already prints option/usage info that's automatically generated on parse failure, and I don't see a hook to customize that.

While looking into this, however, I did notice that the "stand-alone" main()s in src/test/scala/org/hammerlab/guacamole/main were not being packaged into our JAR when we run `mvn package`, making them seemingly impossible to build/run, so I moved them back under src/main/scala. I didn't see an obvious way to reconfigure Maven to allow them to be built as test JARs.